base/applications/games/winmine/lang/ar-AR.rc

```diff
@@ -0,0 +1,83 @@
+/*
+ * PROJECT:    WineMine
+ * LICENSE:    GPL - See COPYING in the top level directory
+ * FILE:       base/applications/calc/lang/ar-AR.rc
+ * PURPOSE:    Arabic Language File for WineMine
+ * TRANSLATOR: Mahmoud Ahmed
+ * ADDED:      3/22/2024
+ * LAST UPDATED: 3/22/2024
```

Comment:
Also deprecated style. Current template for header files can be found here: https://reactos.org/wiki/Coding_Style#File_Structure